scripts/create_all_packages.sh

```diff
@@ -80,6 +80,11 @@ if [ "${DRYRUN}zz" == "zz" ]; then
     # Change permissions on source files to match those expected by the server
     chmod 644 build/*
     CURRENT_DATE=$(date +'%Y-%m-%d')
+    # Rename files to include the date and remove extraneous info so that permalinks can be generated
+    for file in build/*; do
+        target=$(sed -E "s/-[0-9.]+commit[^.]+/_$CURRENT_DATE/" <<<"$file")
+        mv "$file" "$target"
```

Comment:
It might be a good idea to handle the case of `$target` already existing.
It's true that our nightly build runs once a day normally. But we might want/need to run it several times.
Using the `--backup` option of `mv` command-line might be enough for this rare use-case

Reply:
Thanks for the suggestion! In my version of Linux (Ubuntu on WSL), in the bash terminal, a `mv` command overwrites silently (without error) any existing target, unless I add the `-no-clobber` option. Wouldn't it be enough that it's overwritten, or do you want explicitly to keep any builds made when re-running the script? If you think we should keep a backup of any overwritten versions, then I'll add `--backup` as you suggest.

Reply:
I've added backup option.